docs/features/datastores/2_Using Datastores/2_creating-datastores.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Creating Datastores
+sidebar_label: Creating Datastores
+---
+# Create Datastores
+
+Datastores can be created from a number of sources (on both the CLI and UI):
+
+- Uploading files stored on your computer
+- From an HTTP URL which downloads a `.zip` or `.tar.gz` file. These files are
+  automatically extracted by the Grid Platform before they are attached to a Run or
+  Session
+- From a public S3 Bucket using the syntax `s3://bucket-name/path/` 
+
+## Uploading Files from a Computer
+
+### Small datasets
+
+You can use the UI to create Datastores for datasets smaller than 1GB (files or folder).
+We have noticed that when the Datastore sizes are 1GB+ you start to hit the browser limit
+for uploading data. In such situations we advise using the CLI to create Datastores.
+
+Select the file or folder and click upload.
+
+![](/images/datastores/ds_upload.gif)
+
+:::note 
+
+You can also use the CLI for uploading these datastores!
+
+:::
+
+### Large datasets (1 GB+)
+
+For datasets larger than 1 GB, use the CLI.
+
+
+:::note 
+
+If you have a dataset that is 1Gb+ and poor-average internet connection, we suggest starting an Interactive Session and creating the Datastore from there. Internet speed is much faster in Interactive Sessions, so upload times will be shorter. 
+
+:::
+
+First, install the grid CLI and login:
+
+```bash
+pip install lightning-grid --upgrade
+grid login
+```
+
+Next, use the Datastores command to upload any folder:
+
+```bash
+grid datastore create ./imagenet_folder/ --name imagenet
```

Comment:
although this works, dose not follow the CLI convention 

s/
grid datastore create ./imagenet_folder/ --name imagenet/
grid datastore create --name imagenet ./imagenet_folder/ 
/